src/api/users/routes.py

```diff
@@ -0,0 +1,25 @@
+"""users endpoint module."""
+
+from typing import Any
+
+from flask_restx import Namespace, Resource
+
+from src.users.models import User
+
+
+api = Namespace("users", description="Get users id and name")
```

Comment:
I'm not really familiar with best practices for flask-restx, but as far as I know, they use `ns` to name `Namespace` objects. It's analagous to `bp` for `Blueprint`.

Another reason, why it's probably would be better to name it `ns` is this import inside `src/api/routes.py`:
```python
from src.api.users.routes import api as users
```
here `import api as users` looks a little bit weird. So it'd be better if we imported and used it like that:
```python
import src.api.users.routes.ns

...

api.add_namespace(src.api.users.routes.ns)
```

The same as we do for registering blueprints